GMX-Bots/Detect-many-closing/src/agent.ts

```diff
@@ -0,0 +1,57 @@
+import { NetworkData, networkData } from "./network";
+import { ethers, Finding, getEthersProvider, HandleTransaction, Log, TransactionEvent } from "forta-agent";
+import { NetworkManager } from "forta-agent-tools";
+
+import utils from "./utils";
+
+const networkManager = new NetworkManager(networkData);
+
+export const provideInitialize = () => {
+  return async () => {
+    const provider = getEthersProvider();
+    await networkManager.init(provider);
```

Comment:
It would make more sense to pass the provider as a parameter and use the one you are importing from utils.